projects/rocfft/clients/tests/accuracy_test.cpp

```
@@ -450,61 +444,7 @@ void apply_store_callback(const fft_params& params, std::vector<hostbuf>& output
-        {
-        case fft_precision_half:
-        {
-            const size_t elem_size = sizeof(rocfft_complex<rocfft_fp16>);
-            for(auto& buf : output)
-            {
-                const size_t num_elems = buf.size() / elem_size;
-
-                auto output_begin = reinterpret_cast<rocfft_complex<rocfft_fp16>*>(buf.data());
-                for(size_t i = 0; i < num_elems; ++i)
-                {
-                    auto& element = output_begin[i];
-                    if(params.scale_factor != 1.0)
-                        element = element * params.scale_factor;
-                }
-            }
-            break;
-        }
-        case fft_precision_single:
-        {
-            const size_t elem_size = sizeof(rocfft_complex<float>);
-            for(auto& buf : output)
-            {
-                const size_t num_elems = buf.size() / elem_size;
-
-                auto output_begin = reinterpret_cast<rocfft_complex<float>*>(buf.data());
-                for(size_t i = 0; i < num_elems; ++i)
-                {
-                    auto& element = output_begin[i];
-                    if(params.scale_factor != 1.0)
-                        element = element * params.scale_factor;
-                }
-            }
-            break;
-        }
-        case fft_precision_double:
-        {
-            const size_t elem_size = sizeof(rocfft_complex<double>);
-            for(auto& buf : output)
-            {
-                const size_t num_elems = buf.size() / elem_size;
-
-                auto output_begin = reinterpret_cast<rocfft_complex<double>*>(buf.data());
-                for(size_t i = 0; i < num_elems; ++i)
-                {
-                    auto& element = output_begin[i];
-                    if(params.scale_factor != 1.0)
-                        element = element * params.scale_factor;
-                }
-            }
-            break;
-        }
-        }
+        // planar wouldn't run callbacks
```

Maybe better to throw a runtime exception instead? I feel like this could trigger much head-scratching down the road as [`fft_params::valid()`](https://github.com/ROCm/rocm-libraries/blob/6709ec0ad2552bf17a5bc43eaaa612cd7a7e34d1/projects/rocfft/shared/fft_params.h#L1693) does not seem to rule such configurations out at the moment.